### PR TITLE
change reference from members to contacts

### PR DESCRIPTION
**Pull request by chris18890**
repoint URL references for members to contacts - it's possible to have the member module disabled, but these links create a dependency on a module that's loaded after the contact & user modules

### CODE REVIEW

**Comment by chris18890** [comment]
@drid @ramgarden @elplatt can I get a review on this?

**Comment by ramgarden** [comment]
Still need a review on this one?  Or does it need to wait for the big merge
to release?

**Comment by chris18890** [comment]
@ramgarden if you're happy to review I'll merge it into dev now, then when the other PRs are ready I'll do a merge into Main & create a release :)

**Review by ramgarden — APPROVED** [review]

**Pull request merged**
